canonical-websites/jaas.ai
# Restructure the application

## Done
Restructured the application to follow the patterns from snapcraft.io

## QA
- Run `./run`
- Go to http://0.0.0.0:8029/
- Check the page loads
- Click around the pages
- Go to a charm/bundle details page (i.e. http://0.0.0.0:8029/wordpress) and check the page displays HELLO

Fixes https://github.com/ubuntudesign/juju-squad/issues/358
